src/patternfly/components/Card/examples/Card.md

```diff
@@ -899,6 +899,7 @@ A card is a generic rectangular container that can be used to build other compon
 | `.pf-v6-c-card` | `<div>` | Creates a card component.  **Required** |
 | `.pf-v6-c-card__title` | `<div>` | Creates a card title container. |
 | `.pf-v6-c-card__title-text` | `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, `<h6>`, `<div>` | Creates a card title text element. |
+| `.pf-v6-c-card__description` | Creates a card description element within the card title. |
```

Comment:
```suggestion
| `.pf-v6-c-card__description` | Creates the description of a card. |
```

I would just say "Creates the description of a card." I don't think we want to put it in the `<CardTitle>` because 1) we don't know what users will put in there and 2) the card title doesn't span the width of the card. Here's a screenshot of the description in `<CardTitle>`.

<img width="1010" height="231" alt="Image" src="https://github.com/user-attachments/assets/dc86e467-d5cc-4736-aa8b-1371822a6e38" />

We probably don't even want to put it in `<CardHeader>` because that is a flex row and doesn't wrap by default. IMO it would be a sibling of `<CardBody>` and come before any card body elements.